templates/application.scss

```diff
@@ -1,6 +1,6 @@
 @charset "utf-8";
 
-@import "normalize-rails";
+@import "normalize-css";
```

Comment:
This is an error and will be fixed once https://github.com/thoughtbot/guides/pull/486 gets merged.